lightning/src/ln/functional_tests.rs

```diff
@@ -9888,11 +9888,13 @@ fn do_test_max_dust_htlc_exposure(dust_outbound_balance: bool, exposure_breach_e
 	let chanmon_cfgs = create_chanmon_cfgs(2);
 	let mut config = test_default_channel_config();
 	config.channel_config.max_dust_htlc_exposure = if multiplier_dust_limit {
-		// Default test fee estimator rate is 253 sat/kw, so we set the multiplier to 5_000_000 / 253
-		// to get roughly the same initial value as the default setting when this test was
-		// originally written.
-		MaxDustHTLCExposure::FeeRateMultiplier(5_000_000 / 253)
-	} else { MaxDustHTLCExposure::FixedLimitMsat(5_000_000) }; // initial default setting value
+		// Default test fee estimator rate is 253 sat/kw plus 2530
+		// sat/kw buffer added by `get_dust_buffer_feerate`, so we
+		// set the multiplier to 5_002_500 / 253 to get roughly
+		// the same initial value as the default setting when this
+		// test was originally written.
```

Comment:
umm why do you expect it to be 2783? shouldn't our goal here be to calculate the multiplier to 2783? 

My main motivation here was to raise the total dust limit taking into account the additional 1 sat/vb that is now added to the fee rate.  The test below fills out the full dust capacity and then tries to route payments expecting them to fail but they were successful in some cases. 
Maybe its getting tricky here because its not very clear to me how the initial 5_000_000 / 253 was decided on and passing the test ends up being fixing the numbers.